src/pages/split-condition/SplitCondition.tsx

```diff
@@ -1,90 +1,142 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useCallback } from 'react'
 import { BigNumber } from 'ethers/utils'
-import { useForm, Controller } from 'react-hook-form'
+import { useForm } from 'react-hook-form'
 import { Token } from '../../config/networkConfig'
 import { SetAllowance } from '../../components/common/SetAllowance'
-import { BigNumberInputWrapper } from '../../components/common/BigNumberInputWrapper'
 import { ZERO_BN } from '../../config/constants'
-import { range } from '../../util/tools'
+import { trivialPartition } from '../../util/tools'
 import { Remote } from '../../util/remoteData'
-import { ConditionalTokensService } from '../../services/conditionalTokens'
-
-const bytesRegex = /^0x[a-fA-F0-9]{64}$/
-const NULL_PARENT_ID = '0x0000000000000000000000000000000000000000000000000000000000000000'
-
-type Form = {
+import { InputAmount } from './InputAmount'
+import { InputPosition } from './InputPosition'
+import { SelectCollateral } from './SelectCollateral'
+import { InputCondition } from './InputCondition'
+import { fetchPosition_position } from 'types/generatedGQL'
+import { ERC20Service } from 'services/erc20'
+import { useWeb3Connected } from 'contexts/Web3Context'
+
+export const bytesRegex = /^0x[a-fA-F0-9]{64}$/
+export const NULL_PARENT_ID = '0x0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
I think this should exist in the constant file (Line 17 and 18)

Reply:
This is done in your PR of condition details, should be solved after merge.